glowbot/commands.py

```diff
@@ -35,6 +35,8 @@ async def register(self, ctx: discord.ApplicationContext, steam64: Option(
         await ctx.defer(ephemeral=True)
 
         # See if the user already has an entry
+        # todo: need to validate steam64 is an integer of correct form...
```

Comment:
I was thinking rudimentary validation until the code validates via steam api, even it just ensures there are the correct number of digits in the interger.
I was able to break everything very easily by doing
/hll register 1
/hll register 2
/hll register 3

Reply:
Looking at this again, I think it would be easiest to allow registrations with steamid64s that are already in the db. This way the code **knows** that the steamid64 is valid since it came from the game itself.

Is there a compelling reason to allow ppl to register before they have seeded on the server?

Or maybe have the bot record steamids even when the server is not seeding. Then there is a record of all past players to validate against. It seems that the number of player wanting to register their steamid before ever playing on the server would be very small.